perf/istio-install/automtls.yaml

```diff
@@ -3,6 +3,8 @@ kind: IstioControlPlane
 spec:
   values:
     global:
+      sidecarInjectorWebhook:
+        rewriteAppHTTPProbe: true
```

Comment:
Why is this needed for the mtls overhead test?

Reply:
turning on mtls blocks health check. this solves the problem.

does no harm anyway,, only impacting sidecar injector

Reply:
Just to double check, is your automtls.yaml overlays on top of the operator built-in default profile or the operator_default.yaml defined here

Reply:
I'm using whatever defined in the `setup_istio_operator.sh`, open to update that shell library.

https://github.com/istio/tools/blob/f9fdb1d49b55df4ca63447aaab16398c68024fa2/perf/istio-install/setup_istio_operator.sh#L63

Propogates as

```
go run ./cmd/mesh.go manifest apply -f "${CR_FILENAME}" --force=true --set defaultNamespace=${defaultNamespace}
```

THe `CR_FILENAME` is replaced, that might be not overlay?